demos/00-helpers/src/helpers.js

```diff
@@ -1,3 +1,26 @@
+import { setCORS } from "google-translate-api-browser";
+const translate = setCORS("http://cors-anywhere.herokuapp.com/");
+
+function schedule_a_mock_reply(message_text, callback)
+{
+	message_text = g_scm2host(message_text);
+	console.log("translating " + message_text.toString() + "...");
+	translate(message_text, { to: "en" })
+		.then(res => {
+			console.log('ok, translation is:' + res.text);
+			g_scm_call(callback,[g_host2scm(res.text)]);
+		})
+		.catch(err => {
+			console.log('translate err');
+			console.error(err);
+			g_scm_call(callback,err.toString())
```

Comment:
I am not sure what you want to do with `g_scm_call` if you want to retrieve the result scheme side that is not the way to go... see the finale review note.